tools/block-generator/run_runner.sh

```diff
@@ -13,7 +13,7 @@ fi
 POSTGRES_CONTAINER=generator-test-container
 POSTGRES_PORT=15432
 POSTGRES_DATABASE=generator_db
-CONFIG=${2:-"$(dirname $0)/test_config.yml"}
+CONFIG=${4:-"$(dirname $0)/test_config.yml"}
```

Comment:
This script needs a little extra attention if we want to keep using it. Using positional arguments means they're all required and having so many of them gets confusing. As it is, it would be easy to run the script incorrectly.

$1 - conduit binary
$2 - db-round (required if you want a config template)
$3 - genesis file (required if you want a config template)
$4 - config template (optional)

Reply:
My follow up has a python version of this that's a bit more flexible.